src/calculateRentalCost.js

```diff
@@ -3,8 +3,31 @@
  *
  * @return {number}
  */
+
+const minDiscount = 20;
+const maxDiscount = 50;
+
+let discountSum = 0;
+const daysPrice = 40; /* $ */
+
 function calculateRentalCost(days) {
-  // write code here
+  const totalSum = days * daysPrice;
+
+  if (days < 3) {
+    return totalSum;
+  }
+
+  if (days === 3 || days < 7) {
+    discountSum = totalSum - minDiscount;
+
+    return discountSum;
+  }
+
+  if (days === 7 || days > 7) {
```

Comment:
The condition `days === 7 || days > 7` is redundant. It can be simplified to `days >= 7` to apply the maximum discount for rentals of 7 or more days.